### PR TITLE
Fix add existing space not showing any spaces

### PR DESCRIPTION
**Pull request by c-cal**
Fixes https://github.com/vector-im/element-web/issues/21087

Type: defect

Signed-off-by: Charlie Calendre [c-cal@watcha.fr](mailto:c-cal@watcha.fr)

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix add existing space not showing any spaces ([\#7801](https://github.com/matrix-org/matrix-react-sdk/pull/7801)). Fixes vector-im/element-web#21087. Contributed by @c-cal.

Preview: https://pr7801--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by c-cal** [comment]
I may have missed something, but I don't see the point between the comment and the code I deleted with the current UI

File: src/components/structures/SpaceRoomView.tsx [hunk 1]
```diff
@@ -614,7 +614,20 @@ const SpaceAddExistingRooms = ({ space, onFinished }) => {
             filterPlaceholder={_t("Search for rooms or spaces")}
             onFinished={onFinished}
             roomsRenderer={defaultRoomsRenderer}
-            spacesRenderer={defaultSpacesRenderer}
+            spacesRenderer={() => (
```
Comment (t3chguy) [inline src/components/structures/SpaceRoomView.tsx:617]: sorry I didn't realise which call-site this was, I think there should be no spacesRenderer here and it should just show rooms to add instead

Comment (c-cal) [reply src/components/structures/SpaceRoomView.tsx]: I think so... done

**Review by t3chguy — APPROVED** [review]
LGTM thanks

**Pull request merged**